rust/release/src/main.rs

```diff
@@ -432,3 +468,19 @@ Overview:"#
     }
     Ok(())
 }
+
+fn update_package_version(path: &PathBuf, version: &Version) -> Result<(), Error> {
+    // Read the toml
+    let cargo_toml_contents = if !path.is_file() {
+        bail!("{:?} is not a file", path);
```

Comment:
nitpick: this logic should have been covered by `std::fs::read_to_string`

Reply:
👍